app.rb

```diff
@@ -27,18 +27,18 @@
   on post do
     on "create" do
       name = req.params["name"]
-      email = req.params["email"]
-      discord = req.params["discord"]
+      first-album-year = req.params["year"]
+      my-rank = req.params["rank"]
```

Comment:
Does this work? I don't _think_ Ruby allows putting dashes in variable names. In fact, your syntax highlighter in your editor may have gotten confused about these variables. 

I think Ruby would interpret these lines as subtraction. For example, on line 31, it probably thinks you are attempting to subtract a variable called`rank` from a variable called `my`. 

The conventional way to write variables in Ruby is with underscores: `first_album_year` and `my_rank`.